superset/models/core.py

```diff
@@ -377,7 +377,9 @@ class Tag(Model):
 
     __tablename__ = 'tags'
     id = Column(Integer, primary_key=True)
-    tag_name = Column(String(255))
+    __table_args__ = (UniqueConstraint('tag_name'),)
```

Comment:
Just wondering if you need both the `unique=True` arg below and this one? I added the `unique=True` and regenerated the DB migration script to add `     sa.UniqueConstraint('tag_name'),`